components/organisms/ProfileTabBody.tsx

```diff
@@ -0,0 +1,193 @@
+                <ProfileRecordItem
+                  icon={EthTransparentSVG}
+                  key={key}
+                  label={key}
+                  text={value}
+                />
+              </Skeleton>
+            ))}
+          </div>
+        </div>
+      )}
+      <div className="flex flex-col gap-4">
+        <Skeleton>
+          <h3 className="font-semibold text-base">Ownership</h3>
+        </Skeleton>
+
+        <div className="grid grid-cols-2 gap-4">
+          <Skeleton>
+            <ProfileRecordItem
+              icon={CogSVG}
+              label="manager"
+              text={domainData?.owner ?? ""}
+            />
+          </Skeleton>
+
+          <Skeleton>
+            {domainData?.owner && (
+              <ProfileRecordItem
+                icon={HeartSVG}
+                label="owner"
+                text={domainData?.owner}
+              />
+            )}
+          </Skeleton>
+
+          <Skeleton>
+            {expiryDate !== undefined && (
+              <ProfileRecordItem
+                icon={CalendarSVG}
+                label="expiry"
+                text={formatDate({
+                  unixTimestamp: millisencondsToSeconds(expiryDate),
+                })}
+              />
+            )}
+          </Skeleton>
+          <Skeleton>
+            <ProfileRecordItem
+              icon={EthTransparentSVG}
+              label="parent"
+              text="ETH"
+            />
+          </Skeleton>
+        </div>
+      </div>
+
+      <div className="flex flex-col gap-4">
+        <Skeleton>
+          <h3 className="font-semibold text-base">Resolver</h3>
+        </Skeleton>
+        <div className="grid grid-cols-2 gap-4">
+          <Skeleton>
+            <div className="flex justify-between items-center gap-4 px-2 py-2 rounded-md bg-gray-50 overflow-hidden">
+              <div className="flex items-center gap-4">
+                <div className="flex gap-4 p-2 rounded-md bg-blue-100">
+                  <DatabaseIcon className="h-5 w-5 text-blue-500" />
+                </div>
+
+                {resolver?.address && (
+                  <p className="whitespace-nowrap truncate">
+                    {formatHexAddress(resolver?.address)}
+                  </p>
+                )}
+              </div>
+
+              {showEditButton && (
+                <div>
+                  <Button
+                    onClick={() => {
+                      setEditResolverModalOpen(true);
+                    }}
+                    size="small"
+                  >
+                    Edit
+                  </Button>
+                </div>
+              )}
+            </div>
+          </Skeleton>
+        </div>
+      </div>
+      {resolver?.address && (
+        <Modal open={editResolverModalOpen} onDismiss={() => {}}>
+          <EditResolverModalContent
+            currentResolverAddress={resolver?.address}
+            name={name as string}
```

Comment:
Can you add "name" as a prop so it is passed from `page.tsx` to this component?

It seems more reliable to do so as the time comes,

This component shouldn't be used in another context but it also wouldn't work since it requires one query prop. Receiving it in props shall make this explicit.

Reply:
@FrancoAguzzi I'll create an issue for that